lms/static/sass/discussion/_discussion.scss

```diff
@@ -6,23 +6,11 @@ body.discussion {
 
   .course-tabs .right {
     @include float(right);
-
-    .new-post-btn {
-      @include blue-button;
-      @include margin-right(4px);
-    }
-
-    .new-post-icon {
-      @include margin-right(7px);
-      font-size: 16px;
-      vertical-align: middle;
-      color: $white;
-    }
   }
 
   .edit-post-form {
     @include clearfix();
-    @include box-sizing(border-box);
+    box-sizing: border-box;
```

Comment:
I think the `@include` uses Bourbon and automatically spits out different rules for various browsers.